package/MDAnalysis/coordinates/TPR.py

```diff
@@ -0,0 +1,85 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+
+from . import base
+from ..lib import util
+from .timestep import Timestep
+import MDAnalysis.topology.tpr.utils as tpr_utils
+import MDAnalysis.topology.tpr.setting as S
+
+import logging
+
+logger = logging.getLogger("MDAnalysis.coordinates.TPR")
+
+import numpy as np
+
```

Comment:
Needs some docs to be included in the sphinx doc tree. 

Include a versionadded 2.10.0.